### PR TITLE
Introducing new stream to manage sms

### PR DESCRIPTION
**Pull request by somindatommy**
## Purpose
Currently, notifications are managed by using the stream `id_gov_notify_stream_1.0.0.json`. Ideally, there should be separate streams to manage emails, SMS etc. This PR introduces a new stream to manage SMS based notifications.

Associated with PR : https://github.com/wso2-extensions/identity-governance/pull/324

### CODE REVIEW

File: features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml [hunk 1]
```diff
@@ -22,6 +22,11 @@
   -->
 
 <configurations>
+    <configuration type="smsaccountconfirmation" display="smsaccountconfirmation" locale="en_US" emailContentType="text/html">
```
Comment (pulasthi7) [inline features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Can we avoid this attribute 'emailContentType'?
And if not possible change the content type to text/plain

Comment (somindatommy) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: The attribute is referred to by the management console. Therefore, I think we cannot remove it.
+1 for changing the content type.

Comment (somindatommy) [reply features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml]: Fixed : 5a20b3e2f24c3721024b8a9d5762fc385b46fa1c

File: features/org.wso2.carbon.event.handler.notification.server.feature/resources/p2.inf [hunk 2]
```diff
@@ -5,4 +5,5 @@ org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../../r
 org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../../repository/deployment/server/eventstreams); \
 org.eclipse.equinox.p2.touchpoint.natives.mkdir(path:${installFolder}/../../../repository/deployment/server/eventpublishers); \
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.event.handler.notification.server_${feature.version}/id_gov_notify_stream_1.0.0.json,target:${installFolder}/../../deployment/server/eventstreams/id_gov_notify_stream_1.0.0.json,overwrite:true);\
+org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.event.handler.notification.server_${feature.version}/id_gov_sms_notify_stream_1.0.0.json,target:${installFolder}/../../deployment/server/eventstreams/id_gov_sms_notify_stream_1.0.0.json,overwrite:true);\
 org.eclipse.equinox.p2.touchpoint.natives.copy(source:${installFolder}/../features/org.wso2.carbon.event.handler.notification.server_${feature.version}/EmailPublisher.xml,target:${installFolder}/../../deployment/server/eventpublishers/EmailPublisher.xml,overwrite:true);\
```
Comment (thanujalk) [inline features/org.wso2.carbon.event.handler.notification.server.feature/resources/p2.inf]: add a new line at the end of the file.

Comment (somindatommy) [reply features/org.wso2.carbon.event.handler.notification.server.feature/resources/p2.inf]: Fixed in https://github.com/wso2-extensions/identity-event-handler-notification/pull/112

**Pull request merged**